defaults/main.yml

```diff
@@ -6,41 +6,41 @@ graylog_java_repo:                 "deb http://ppa.launchpad.net/webupd8team/jav
 graylog_java_src_repo:             "deb-src http://ppa.launchpad.net/webupd8team/java/ubuntu xenial main"
 graylog_java_repo_keyserver:       "keyserver.ubuntu.com"
 graylog_java_repo_key:             "EEA14886"
-graylog_version:                   "2.4"
+graylog_version:                   "3.0"
 graylog_package_state:             "present"
-graylog_apt_deb_url:               "https://packages.graylog2.org/repo/packages/graylog-{{ graylog_version }}-repository_latest.deb"
+graylog_apt_deb_url:               "https://packages.graylog2.org/repo/packages/graylog-3.0-repository_latest.deb"
 graylog_manage_apt_repo:           True
-graylog_yum_rpm_url:               "https://packages.graylog2.org/repo/packages/graylog-{{ graylog_version }}-repository_latest.rpm"
+graylog_yum_rpm_url:               "https://packages.graylog2.org/repo/packages/graylog-3.0-repository_latest.rpm"
 graylog_manage_yum_repo:           True
-graylog_java_oracle_installer_key: "oracle-java8-installer"
-graylog_java_oracle_license_key:   "accepted-oracle-license-v1-1"
-# Pin to compatible ElasticSearch version to avoid issues if newer repositories are present, e.g. for Logstash 6.X.
-graylog_es_debian_pin_version:     "5.*"
+graylog_es_debian_pin_version:     "6.*"
 
 # General
 graylog_is_master:            True
 graylog_node_id_file:         "/etc/graylog/server/node-id"
 graylog_password_secret:      "2jueVqZpwLLjaWxV"
 graylog_root_username:        "admin"
 graylog_root_password_sha2:   "8c6976e5b5410415bde908bd4dee15dfb167a9c873fc4bb8a81f6f2ab448a918"
+graylog_bin_dir:              "/usr/share/graylog-server/bin"
+graylog_data_dir:             "/usr/share/graylog-server/data"
```

Comment:
The default `data_dir` in the OS packages is `/var/lib/graylog-server`. I think we should use that here as well.

See:

https://github.com/Graylog2/fpm-recipes/blob/47b055e404660fc91073ef9c56c5bc1043a5fd83/recipes/graylog-server/patches/graylog-server.conf.patch#L13-L14